src/config/client-config.c

```diff
@@ -453,9 +461,12 @@ int Read_Client_Enrollment(XML_NODE node, agent * logr){
             os_free(target_cfg->centralized_group);
             os_strdup(node[j]->content, target_cfg->centralized_group);
         } else if (strcmp(node[j]->element, xml_agent_addr) == 0) {
-            if (OS_IsValidIP(node[j]->content, NULL) != 0) {
+            if (OS_IsValidIP(node[j]->content, NULL) == 1) {
```

Comment:
This configuration, Does not allow CIDR or Netmask?
OS_IsValidIP return 2 when it find CIDR or netmask.

Reply:
It should be a single IP, so it is not allowed a subnet.